<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>numeroPrimo</class>
 <widget class="QDialog" name="numeroPrimo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>NUMERO PRIMOS</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultPrimos">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>249</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLineEdit" name="txtNumPrimo"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pbAceptarNumPrimo">
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Ingrese un numero para saber si es primo o no:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
